DbgProvider/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Buffers" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.3.0" newVersion="4.0.3.0" />
```

Comment:
>4.0.3.0 [](start = 66, length = 7)

I assume that all the app.config/packages.config goo was done by VS tooling, but... do you have any idea why this would be redirecting to version 4.0.3.0, but the version specified by the packages.config is 4.5?

I wonder how much of this app.config stuff is actually necessary--sometimes I find that VS throws crud into my solution that turns out to not be important at all. Does this fall in that category?